config/_default/params.toml

```
@@ -16,6 +16,7 @@ enableCodeCopy = true
 # robots = ""
 
 [footer]
+  showMenu = true
```

Should I delete this one ? what about the others ?

---

Maybe we can comment it out instead

```suggestion
 # showMenu = true
```